diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/generation/Generation.kt

```diff
@@ -47,6 +60,118 @@ private fun generateWarningNames() {
     kotlinFile.writeTo(File("diktat-rules/src/main/kotlin"))  // fixme: need to add it to pom
 }
 
+@Suppress("LoopWithTooManyJumpStatements", "LongMethod", "ComplexMethod", "NestedBlockDepth")
+private fun generateCodeStyle() {
```

Comment:
As I've said in https://github.com/cqfn/diKTat/pull/568#discussion_r535095854, it's not a very good idea to mix docs generation with codegen. Also, we don't need to regenerate docs on every change and do it manually now. Let's move this logic closer to docs and then refactor and unify all of those scripts.